packages/ckeditor5-table/src/commands/mergecellcommand.js

```diff
@@ -108,7 +107,9 @@ export default class MergeCellCommand extends Command {
 
 			// Remove empty row after merging.
 			if ( !removedTableCellRow.childCount ) {
-				removeEmptyRow( removedTableCellRow, writer );
+				const tableUtils = this.editor.plugins.get( 'TableUtils' );
+
+				tableUtils.removeRows( table, { at: table.getChildIndex( removedTableCellRow ), batch: writer.batch } );
```

Comment:
```suggestion
				tableUtils.removeRows( table, { at: removedTableCellRow.index, batch: writer.batch } );
```

A bit shorter. Basically the same and